files/en-us/web/css/css_flexible_box_layout/ordering_flex_items/index.html

```diff
@@ -100,7 +100,7 @@ <h2 id="The_order_property">The order property</h2>
 
 <p>Flex items have a default <code>order</code> value of <code>0</code>, therefore items with an integer value greater than 0 will be displayed after any items that have not been given an explicit <code>order</code> value.</p>
 
-<p>You can also use negative values with order, which can be quite useful. If you want to make one item display first, and leave the order of all other items unchanged, you can give that item an order of <code>-1</code>. As this is lower than 0 the item will always be displayed first.</p>
+<p>You can also use negative values with an order, which can be quite useful. If you want to make one item display first and leave the order of all other items unchanged, you can give that item order of <code>-1</code>. As this is lower than 0 the item will always be displayed first.</p>
```

Comment:
order was correct, but didn't have the correct typo.
```suggestion
<p>You can also use negative values with <code>order</code>, which can be quite useful. If you want to make one item display first and leave the order of all other items unchanged, you can give that item order of <code>-1</code>. As this is lower than 0 the item will always be displayed first.</p>
```